transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollChannelOption.java

```diff
@@ -26,6 +26,9 @@
 
     public static final ChannelOption<Boolean> SO_REUSEPORT = valueOf("SO_REUSEPORT");
 
+    public static final ChannelOption<EpollDomainSocketReadMode> DOMAIN_SOCKET_READ_MODE =
```

Comment:
Let me call it DomainSocketReadMode for now. I will do the repackaging stuff in another PR... sound good ?